gandalf.js

```diff
@@ -1,35 +1,37 @@
-/**
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.